pkgs/development/python-modules/colormath/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, networkx
+, nose
+, numpy
+, lib
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "colormath";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "gtaylor";
+    rev = "3.0.0";
+    repo = "python-colormath";
+    sha256 = "1nqf5wy8ikx2g684khzvjc4iagkslmbsxxwilbv4jpaznr9lahdl";
+  };
+
+  # Are there no tests?
+  checkInputs = [ nose ];
+  checkPhase = "nosetests";
+
+  propagatedBuildInputs = [ networkx numpy ];
```

Comment:
generally propagatedBuildInputs is ordered before check inputs as they get referenced first during the build

```
  propagatedBuildInputs = [ networkx numpy ];

  checkInputs = [ nose ];
  checkPhase = "nosetests";
```